source/databases-collections.txt

```diff
@@ -119,31 +119,39 @@ initial collection.
 
 To create a new database:
 
-1. Hover over the connection for the deployment where you want your
+#. Hover over the connection for the deployment where you want your
    database to exist.
 
 #. Click the :icon-fa5:`plus` icon that appears.
 
-#. In the prompt, enter a name for your new database.
+#. A JavaScript file populates in a new tab with const 
```

Comment:
[suggestion]

I think we can be clearer here about who the actor here is in this step and remove passive voice. Consider something like:

"{+vsce+} opens a new tab with ``const`` variables for database and collection names."